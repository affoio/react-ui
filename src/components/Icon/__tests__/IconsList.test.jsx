import React from 'react';
import { render } from 'enzyme';
import CloseEyeIcon from '../CloseEyeIcon';
import OpenEyeIcon from '../OpenEyeIcon';
import SearchIcon from '../SearchIcon';

describe('Icons snapshots', () => {
  it('CloseEyeIcon renders correctly', () => {
    const tree = render(<CloseEyeIcon />);

    expect(tree).toMatchSnapshot();
  });

  it('OpenEyeIcon renders correctly', () => {
    const tree = render(<OpenEyeIcon />);

    expect(tree).toMatchSnapshot();
  });

  it('SearchIcon renders correctly', () => {
    const tree = render(<SearchIcon />);

    expect(tree).toMatchSnapshot();
  });
});
