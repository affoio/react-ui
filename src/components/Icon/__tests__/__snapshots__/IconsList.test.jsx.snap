// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icons snapshots CloseEyeIcon renders correctly 1`] = `
.c0 {
  fill: currentColor;
}

<svg
  class="c0"
  height="19"
  viewBox="0 0 22 19"
  width="22"
>
  <path
    d="M11 4c2.76 0 5 2.24 5 5 0 .65-.13 1.26-.36 1.83l2.92 2.92c1.51-1.26 2.7-2.89 3.43-4.75-1.73-4.39-6-7.5-11-7.5-1.4 0-2.74.25-3.98.7l2.16 2.16C9.74 4.13 10.35 4 11 4zM1 1.27l2.28 2.28.46.46A11.804 11.804 0 0 0 0 9c1.73 4.39 6 7.5 11 7.5 1.55 0 3.03-.3 4.38-.84l.42.42L18.73 19 20 17.73 2.27 0 1 1.27zM6.53 6.8l1.55 1.55C8.03 8.56 8 8.78 8 9c0 1.66 1.34 3 3 3 .22 0 .44-.03.65-.08l1.55 1.55c-.67.33-1.41.53-2.2.53-2.76 0-5-2.24-5-5 0-.79.2-1.53.53-2.2zm4.31-.78l3.15 3.15.02-.16c0-1.66-1.34-3-3-3l-.17.01z"
  />
</svg>
`;

exports[`Icons snapshots OpenEyeIcon renders correctly 1`] = `
.c0 {
  fill: currentColor;
}

<svg
  class="c0"
  height="24"
  viewBox="0 0 24 24"
  width="24"
>
  <path
    d="M12 4.5C7 4.5 2.73 7.61 1 12c1.73 4.39 6 7.5 11 7.5s9.27-3.11 11-7.5c-1.73-4.39-6-7.5-11-7.5zM12 17c-2.76 0-5-2.24-5-5s2.24-5 5-5 5 2.24 5 5-2.24 5-5 5zm0-8c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"
  />
</svg>
`;

exports[`Icons snapshots SearchIcon renders correctly 1`] = `
.c0 {
  fill: currentColor;
}

<svg
  class="c0"
  height="24"
  viewBox="0 0 24 24"
  width="24"
>
  <path
    d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"
  />
</svg>
`;
